### ISSUE TITLE
The model and loaded state dict do not match exactly for H3DNet

### ISSUE THREAD

**Issue by filaPro** [open-mmlab/mmdetection3d#989]
The comment about `convert_votenet_checkpoints.py` is probably also relevant for `H3DNet`. Now I'm getting:

```
The model and loaded state dict do not match exactly

unexpected key in source state_dict: rpn_head.conv_pred.0.conv.weight, rpn_head.conv_pred.0.conv.bias, rpn_head.conv_pred.0.bn.weight, rpn_head.conv_pred.0.bn.bias, rpn_head.conv_pred.0.bn.running_mean, rpn_head.conv_pred.0.bn.running_var, rpn_head.conv_pred.0.bn.num_batches_tracked, rpn_head.conv_pred.1.conv.weight, rpn_head.conv_pred.1.conv.bias, rpn_head.conv_pred.1.bn.weight, rpn_head.conv_pred.1.bn.bias, rpn_head.conv_pred.1.bn.running_mean, rpn_head.conv_pred.1.bn.running_var, rpn_head.conv_pred.1.bn.num_batches_tracked, rpn_head.conv_pred.conv_out.weight, rpn_head.conv_pred.conv_out.bias

missing keys in source state_dict: rpn_head.conv_pred.shared_convs.layer0.conv.weight, rpn_head.conv_pred.shared_convs.layer0.conv.bias, rpn_head.conv_pred.shared_convs.layer0.bn.weight, rpn_head.conv_pred.shared_convs.layer0.bn.bias, rpn_head.conv_pred.shared_convs.layer0.bn.running_mean, rpn_head.conv_pred.shared_convs.layer0.bn.running_var, rpn_head.conv_pred.shared_convs.layer1.conv.weight, rpn_head.conv_pred.shared_convs.layer1.conv.bias, rpn_head.conv_pred.shared_convs.layer1.bn.weight, rpn_head.conv_pred.shared_convs.layer1.bn.bias, rpn_head.conv_pred.shared_convs.layer1.bn.running_mean, rpn_head.conv_pred.shared_convs.layer1.bn.running_var, rpn_head.conv_pred.conv_cls.weight, rpn_head.conv_pred.conv_cls.bias, rpn_head.conv_pred.conv_reg.weight, rpn_head.conv_pred.conv_reg.bias
```

**Comment by ZCMax** [comment]
Hi! thanks for your reminder, for H3DNet, a checkpoint conversion script is also need. We will offer a script `convert_h3dnet_checkpoints.py` for H3DNet checkpoint conversion. The PR will come today.

**Comment by Tai-Wang** [comment]
Fixed by #1007 .
